syntax/output.php

```diff
@@ -79,9 +79,13 @@ public function handle($match, $state, $pos, Doku_Handler $handler) {
      * @return bool If rendering was successful.
      */
     public function render($mode, Doku_Renderer $R, $data) {
+        global $ACT;
         global $ID;
         global $INFO;
         global $REV;
+        if (act_clean($ACT) !== 'show') {
+            return true;
+        }
```

Comment:
Summary of our discussion for documentation:

- `PARSER_HANDLER_DONE` is also called when edit session is parsed/rendered
- We have no reliable way to know which file is currently parsed/rendered
- An alternative might be to test that ID is either identical to the first or last read page and add the instruction if either is true
  - however that is also not a clean solution

=> We will use the currently implemented solution. We will adjust it when we encounter a case where it breaks and/or DokuWiki is changed in a way that gives us the information which page we are currently rendering.